.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
```

Comment:
Can we have something like `<!-- DO NOT IGNORE THIS ISSUE TEMPLATE OR YOUR ISSUE MAY/WILL BE CLOSED -->`.

Applies to other templates too.

Reply:
There is already a note like that, it will always be ignored by some. To effectively enforce this we need to close any issue that is not following issues template (manually)